app/browser/api/ledger.js

```diff
@@ -1138,6 +1138,19 @@ const initSynopsis = (state) => {
   return state
 }
 
+const checkPromotions = () => {
+  if (promotionTimeoutId) {
+    clearInterval(promotionTimeoutId)
+  }
+
+  // get promotions
+  appActions.onPromotionGet()
+
+  promotionTimeoutId = setTimeout(() => {
+    checkPromotions()
+  }, random.randomInt({min: 20 * ledgerUtil.milliseconds.second, max: 24 * ledgerUtil.milliseconds.second}))
```

Comment:
should these be `ledgerUtil.milliseconds.hour`?

Reply:
you are correct, I was using seconds to test code out and forgot to change it back, nice catch